FiveCalls/FiveCalls/IssuesViewController.swift

```diff
@@ -72,12 +79,15 @@ class IssuesViewController : UITableViewController {
     }
     
     override var preferredStatusBarStyle: UIStatusBarStyle {
-        return .lightContent
+        return shouldFetchAllIssues ? .default : .lightContent
     }
     
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         logs = ContactLogs.load()
+        if shouldFetchAllIssues {
+            navigationController?.setNavigationBarHidden(false, animated: animated)
+        }
```

Comment:
I considered this and opted against it. It felt unnecessarily heavy-weight. I am generally pretty averse to inheritance, though.